FreeRTOS/FreeRTOS
# Add base priority get unit test

> !! Note : This pull request should be merged after Kernel PR [#818](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/818)

Description
-----------
* Add uxTaskBasePriorityGet and uxTaskBasePriorityGetFromISR unit test

Test Steps
-----------
Run the unit test ensure the base priority returned is correct.

Checklist:
----------



Related Issue
-----------
This PR is an update for #1093


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
